app/app.js

```diff
@@ -20,12 +23,51 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(express.static(path.join(__dirname, '../public')));
+//app.use(express.bodyParser()); need?
 
 app.get('/', function(req, res, next) {
     console.log('routing to index');
     res.render('index');
     });
 
+app.post('/api/v1/beers', function(req, res, next) {
+    // Construct the brewerydb endpoint to hit
+    endpoint = 'http://api.brewerydb.com/v2/search?q=' + encodeURIComponent(req.body.beer) + '&key=' + secrets.bdb_key
```

Comment:
We do not access `endpoint` until we're already inside of the non-mock response, let's move that there.